canonical/ubuntu.com
# feat: /internet-of-things/appstore redesgin

## Done

- Applied redesign as per [doc](https://docs.google.com/document/d/1CKwmPDDtXOIgw-jSZFwzP-qLMc_rS_dUACpHQXlg0fA/edit?tab=t.0) and [design](https://www.figma.com/design/1CdIT6TV3ASf55GsnQcwXg/Internet-of-Things?node-id=0-1&node-type=canvas&m=dev)

## QA

- View the site locally in your web browser at: https://ubuntu-com-14523.demos.haus/internet-of-things/appstore
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against doc and design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-16630
